### ISSUE TITLE
[BUG] Auto-Pickup and Auto-investigate stopping when there is too much to items around

### ISSUE THREAD

**Issue by Darkstarcracker** [Akebi-Group/Akebi-GC#589]
When there is too much to pick-up and investigate it will stop even if you teleport or switch the auto pickup on and off again it stops working. 

example of this is going to middle of Kannazuka, Tatarasuna it goes crazy and stops

PS. please add investigation Filter I don't want to investigate ores and food when doing artifact runs

**Comment by Taiga74164** [comment]
Can you show video if you got a chance..

About the Investigation filter, there's nothing we can do about it. The game defines it as one category.

**Comment by Darkstarcracker** [comment]
yeah it sucks cause there is a investigation limit hope they remove it cause its effin annoying haha.
ill try to record it

**Comment by Darkstarcracker** [comment]
[https://youtu.be/kMJVwbA9doU](url) heres the video

**Comment by harlanx** [comment]
> [https://youtu.be/kMJVwbA9doU](url)

Hide your UID 💀💀💀

**Comment by Taiga74164** [comment]
thanks

**Comment by darkharrycracker** [comment]
Its a alt anyway

**Comment by Darkstarcracker** [comment]
i deleted the video too haha

**Comment by jdp20205** [comment]
我也遇到过，而且很常会出现

**Comment by CR4Player** [comment]
100% reproducible in Tatarasuna, this acually happens from the first verison support auto-investigate, for me it's not a big problem since i never found any other place has so many investigate points.
